src/gui/mzroll/eicwidget.cpp

```diff
@@ -1081,6 +1081,8 @@ void EicWidget::replot(shared_ptr<PeakGroup> group)
         float rtMin = group->minRt;
         float rtMax = group->maxRt;
         addEICLines(_showSpline, _showEIC, true, rtMin, rtMax);
+        _clearEicPoints();
+        addPeakPositions(group);
```

Comment:
'PeakPositions' must also be added in 'renderPdf' function. Peak-group bubbles are not visible when exported as pdf.

Reply:
This was intentional. They look ugly in the exported PDF. Peak bubbles are meant to convey the quality of each peak which is usually helpful in making decisions when you are within the software itself. The exported PDF is a report for sharing EICs quickly with people who do not want to open the software or have no access to it.